packages/server/src/authenticatorKey/parseFidoPackedKey.js

```diff
@@ -95,23 +109,35 @@ exports.parseFidoPackedKey = (authenticatorKey, clientDataJSON) => {
 
     return {
         fmt: 'packed',
-        publicKey,
+        publicKey: publicKey.toString('base64'),
         counter: authenticatorData.counter,
-        credID: authenticatorData.credID.toString('base64')
+        credID: authenticatorData.credID.toString('base64'),
     };
 };
 
-exports.validateFidoPackedKey = (authenticatorDataBuffer, key, clientDataJSON, base64Signature) => {
+exports.validateFidoPackedKey = (
+    authenticatorDataBuffer,
+    key,
+    clientDataJSON,
+    base64Signature
+) => {
     const authenticatorData = parseAttestationData(authenticatorDataBuffer);
 
-    if (!authenticatorData.flags.up) {
+    // tslint:disable-next-line
+    if (!(authenticatorData.flags.up)) {
         throw new Error('User was NOT presented durring authentication!');
     }
 
-    const clientDataHash = hash('SHA256', Buffer.from(clientDataJSON, 'base64'));
-    const signatureBaseBuffer = Buffer.concat([authenticatorDataBuffer, clientDataHash]);
+    const clientDataHash = hash(
+        'SHA256',
+        Buffer.from(clientDataJSON, 'base64')
+    );
+    const signatureBaseBuffer = Buffer.concat([
+        authenticatorDataBuffer,
+        clientDataHash,
+    ]);
 
-    const publicKey = convertASN1toPEM(Buffer.from(key.publicKey, 'base64'));
+    const publicKey = ConvertASN1toPEM(Buffer.from(key.publicKey, 'base64'));
```

Comment:
Why pascal case ? Isn't it a function ?

Reply:
It's the format name ASN1 (Abstract Syntax Notation One) and PEM (Privacy Enhanced Mail)

Reply:
oups did not see the uppercase